src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,102 @@
+#pause:checked ~ .hand {
+  animation-play-state: running;
+}
+
+.stopwatch {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  height: 500px;
+  width: 500px;
+  margin: 50px auto;
+  background-image: url(../images/stopwatch.png);
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-position: center;
+
+  &__minutes-watch {
+    position: absolute;
+    top: 200px;
+    height: 80px;
+    width: 80px;
+  }
+
+  &__seconds-hand {
+    position: absolute;
+    top: 170px;
+    height: 125px;
+    width: 8px;
+    border-radius: 10px;
+    background-color: red;
+    transform: rotate(0);
+    transform-origin: bottom center;
+    animation: rotate 60s linear infinite paused;
+  }
+
+  &__minutes-hand {
+    position: absolute;
+    top: 215px;
+    height: 25px;
+    width: 3px;
+    border-radius: 10px;
+    background-color: black;
+    transform: rotate(0);
+    transform-origin: bottom center;
+    animation: rotate 3600s linear infinite paused;
+  }
+
+  &__center {
+    position: absolute;
+    top: 285px;
+    width: 20px;
+    height: 20px;
+    border-radius: 50%;
+    background-color: black;
+  }
+
+  &__stop-btn,
+  &__reset-btn {
+    position: absolute;
+    top: 118px;
+    width: 60px;
+    height: 35px;
+    border-radius: 6px;
+    text-transform: uppercase;
+    text-align: center;
+    line-height: 35px;
+    cursor: pointer;
```

Comment:
You can also use `user-select: none;` rule for such controls to avoid text selection. If you want of course :)